### DISCUSSION TITLE
What happened to getAllContents?

### DISCUSSION THREAD

**Discussion by T3rm1** [octokit/octokit.js#2310] [General] [answered]
What happened to the function getAllContents? It should be here: https://octokit.github.io/rest.js/v18#repos-get-contents but it's not.
API documentation: https://docs.github.com/en/rest/repos/contents#get-contents

Search still shows it:
![grafik](https://user-images.githubusercontent.com/5060247/191040822-6225226b-a079-48a7-8ca6-13fe1861847a.png)

**Comment by gr2m** [comment, marked as answer]
The method is called [`.rest.repos.getContent()`](https://octokit.github.io/rest.js/v18#repos-get-content). See https://runkit.com/gr2m/octokit-octokit-js-2310

**Reply by T3rm1** [reply]
Thank you. So the link in the search result is wrong and should be fixed. It has an additional `s` at the end. Not sure why it's now singular. The official GitHub API still uses plural.

**Reply by gr2m** [reply]
I see. Looks like a problem with the website, it's currently not being upgraded due to some Gatsby errors. We will get it fixed eventually, very sorry for the confusion
